velox/expression/FunctionSignature.cpp

```diff
@@ -139,6 +139,48 @@ TypeSignature parseTypeSignature(const std::string& signature) {
   return TypeSignature(baseName, std::move(nestedTypes));
 }
 
+TypePtr signatureToVeloxType(const TypeSignature& typeSignature) {
```

Comment:
curious: Do we support variadic, generic types here ?

Reply:
No, it has no binding.